test/unit/lib/helpers/detainee-attributes.test.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var Detainee = require('../../../../api/lib/helpers/detainee-attributes');
+
+describe('api/lib/helpers/detainee-attributes', () => {
```

Comment:
please name this better, it should at least conform to our existing naming scheme of starting `UNIT` or `INTEGRATION`